popsborder/consignments.py

```diff
@@ -346,6 +401,12 @@ def get_consignment_generator(config):
             items_per_box=config["items_per_box"],
             filename=config["input_file"]["file_name"],
         )
+    elif (generation_method == "input_file") and (
+        config["input_file"]["file_type"] == "variable packaging"
```

Comment:
We use `parameter_based` for method and `variable packaging` here. It is just for consistency since it is key. `input_file` is both value and key, so it is `input_file` in both cases. I don't have a strong opinion, though. In PoPS Core, we ended up supporting all different spellings, but implementing it here would mean some extra work.